src/helm/benchmark/run_expander.py

```diff
@@ -371,6 +371,33 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
         ]
 
 
+class OutputFormatInsturctionsRunExpander(RunExpander):
```

Comment:
Given that we have OpenAIRunExpander and GoogleRunExpander, maybe it makes sense to just MistralRunExpander?  Of course the three could share a subclass or a utility function, but it allows for the possibility of customization...

Reply:
Changed the tag. PTAL?